test/lib/actions/ConfigAction.test.ts

```diff
@@ -398,6 +398,41 @@ describe('ConfigAction tests', () => {
 		});
 	});
 
+	describe('Target/Workspace resolution', () => {
```

Comment:
Validates both the `--workspace` flag in isolation and the `--workspace` flag in conjunction with `--target`.